frontend/src/pages/ForgotPasswordPage.tsx

```diff
@@ -0,0 +1,251 @@
+import React, { useState, useEffect, FormEvent, useMemo } from 'react';
+import { Alert } from 'react-bootstrap';
+import { Trans, useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { ModalContent } from '../components/modals/modalparts/ModalContent';
+import { ModalHeader } from '../components/modals/modalparts/ModalHeader';
+import { Footer } from '../components/Footer';
+import { paths } from '../routers/paths';
+import { Input, errorState } from '../components/forms/FormField';
+import { LoadingSpinner } from '../components/LoadingSpinner';
+import { ReactComponent as MailIcon } from '../icons/mail_icon.svg';
+import { api } from '../api/api';
+import css from './ForgotPassword.module.scss';
+
+const classes = classNames.bind(css);
+
+export const ForgotPasswordPage = () => {
+  const { t } = useTranslation();
+  const defaultDisableTime = 30; // Seconds
```

Comment:
```suggestion
  const defaultDisableSeconds = 30;
```

The comment could be moved to the name, then it would also be visible in each use